CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
 
+## [1.1.5] - 2023-03-29
+### Changed
+
+* Optimized thread locks and expiring cache for the Enhanced Monitoring Plugin. ([PR #356](https://github.com/awslabs/aws-mysql-jdbc/pull/356))
+* Added checking log level before logTrace and logDebug with Messages.getString for better log consistency. ([PR #367](https://github.com/awslabs/aws-mysql-jdbc/pull/367))
+* Refactored reduce map lookups to reduce number of locks required. ([PR #373](https://github.com/awslabs/aws-mysql-jdbc/pull/373))
+* Updated README.md with improvements to the `clusterHostInstancePattern` parameter example.
+* Updated README.md with the latest Maven Central URLs.
+* Updated README.md to reflect that the driver does not support custom RDS clusters.
+* Updated DBeaver documentation in README.md with instructions on how to enable database selectors in the UI.
+* Updated README.md to reflect that Amazon RDS Blue/Green Deployments are not supported.
```

Comment:
Can we group the readme changes for:
1. unsupported items
2. updates/improvements to instructions

Reply:
Done!